solutions/swb-app/src/dataSetRoutes.ts

```diff
@@ -106,7 +103,7 @@ export function setUpDSRoutes(router: Router, dataSetService: DataSetPlugin): vo
 
   // Get dataset
   router.get(
-    '/datasets/:dataSetId',
+    '/projects/:projectsId/datasets/:dataSetId',
```

Comment:
same issue, updating the route without enforcing the project <--> dataset relationship makes the route confusing. If you are using dynamic auth, you may also create an unauthorized disclosure issue.